website/models.py

```diff
@@ -125,12 +125,13 @@ def __str__(self):
         return self.name
 
 class ProjectSelector(models.Model):
+    name = models.CharField(max_length=25, default='')
```

Comment:
what is this?

Reply:
When listing project selections this is a value to name your selector by for easy organization